context.go

```diff
@@ -152,19 +152,24 @@ type ContextVariableList struct {
 	NextPageToken string `json:"next_page_token"`
 }
 
+type ContextListVariablesOptions struct {
+	NextPageToken string `url:"page-token,omitempty"`
```

Comment:
```suggestion
	PageToken string `url:"page-token,omitempty"`
```

Consistency across other `*Options` like `ProjectListPipelinesOptions` etc.

Reply:
Indeed, fixed.